app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
   <div>
-    <p><%= banner.heading %><br>
-    <%= banner.short_description %></p>
+    <h1><%= banner.heading %></h1>
```

Comment:
The reason is that I was following the markup in the original template. I will change it to an h2. I actually thought the HTML outline still stood and that a section would have it's own hierarchy.